packages/eslint-plugin/src/utils/index.ts

```diff
@@ -3,3 +3,4 @@ export { checkIfCompiledExport as validateDefinition } from './create-no-exporte
 export { createNoTaggedTemplateExpressionRule } from './create-no-tagged-template-expression-rule';
 export { CssMapObjectChecker, getCssMapObject } from './css-map';
 export { isCss, isCssMap, isKeyframes, isCxFunction } from './is-compiled-import';
+export { isStyled } from '../rules/no-styled-tagged-template-expression/utils';
```

Comment:
why not move `isStyled` to `is-compiled-import`? importing from a parent directory looks like a code smell imo

Reply:
Hmm that's a good suggestion actually